src/utils/navigation.ts

```diff
@@ -19,6 +19,15 @@ export const getSoldLandFilterBackLink = (): string => {
     return config.LANDING_PAGE_URL;
   }
 };
+
+export const getRemoveBackLink = (req: Request, backLinkUrl: string): string => {
```

Comment:
So this function would need to be called from the 'previousPage' definition for the Update screens below, at least whilst the param URL is still required for journey identification (until data persisted to Mongo).